README.md

```diff
@@ -15,6 +15,12 @@
 
 The Unified Memory Framework (UMF) is a library for constructing allocators and memory pools. It also contains broadly useful abstractions and utilities for memory management. UMF allows users to manage multiple memory pools characterized by different attributes, allowing certain allocation types to be isolated from others and allocated using different hardware resources as required.
 
+**⚠️ Work-In-Progress disclaimer:** <br />
+> Please note that this project is **pre-production software**, it should not be considered complete or fully functional.
+> It has not been fully tested yet (including security testing). This means it is not eligible for Intel® Bug Bounty Program.
```

Comment:
Replace:
> It has not been fully tested yet (including security testing). This means it is not eligible for Intel® Bug Bounty Program.

By:
It has not been fully tested yet (including security testing). It is not recommended to be used in production as part of a larger system. It is not eligible for Intel® Bug Bounty Program.

Reply:
Done.